index.js

```diff
@@ -1,39 +1,49 @@
 /**
  * This strategy is an advanced example of how to customize movements, place blocks, and craft items with the rg-bot package.
  * The Bot will chop wood until it has 100 points-worth of items in its inventory.
- * (Note: Logs and apples are each worth 1 point)
+ * (Note: Coal_Ore and apples are each worth 1 point)
```

Comment:
Apples were a side-effect of collecting wood. Since we don't anticipate a ton of wood-chopping (only enough to make tools), I think it makes sense to remove the apples from this example to simplify it

Reply:
it's an 'advanced' example, so I plan to leave it

I think we could make the advanced example far more advanced in the future

Reply:
In that case, I'd clarify in this comment that apples are a possible byproduct of collecting wood for the pickaxes, so we have a complete picture of why we're bothering with apples